app/assets/stylesheets/trek.scss

```diff
@@ -0,0 +1,20 @@
+.treks {
+  background-color:scale-color($white, $lightness: -5%);
+	-moz-border-radius:3px;
+	-webkit-border-radius:3px;
+	border-radius:3px;
+	display:inline-block;
+	cursor:pointer;
+	padding:5px;
+  margin-bottom: 5px;
+	text-decoration:none;
```

Comment:
Line should be indented with spaces, not tabs<br>Colon after property should be followed by one space